### DISCUSSION TITLE
Control Plane Events information

### DISCUSSION THREAD

**Discussion by dmocek** [edgexfoundry/community#50] [General]
Where can I find more information on Control Plane Events, specifically the format of the CPE JSON?

**Comment by cloudxxx8** [comment]
https://docs.edgexfoundry.org/2.3/design/adr/0024-system-events/
https://docs.edgexfoundry.org/2.3/design/ucr/System-Events-for-Devices/
We only support Device System Event in v2.3, and we are implementing more in v3.0
